inc/spbc-scanner.php

```diff
@@ -832,7 +844,8 @@ function spbc_scanner_scan($direct_call = false, $offset = 0, $amount = 1500, $p
 					NULL
 				);";
 		
-		$success = $wpdb->query($sql);
+		$success = $wpdb->query($sql_hat . $sql_data . $sql_suffix);
+		@$wpdb->query($sql_hat . $sql_data_second . $sql_suffix);
```

Comment:
Suppressing errors is a bad practice.

Reply:
Unfortunately, we don't have any other option. If you want to debate, please, read the corresponding task.